pkg/network/netbinding/netbinding_test.go

```diff
@@ -45,8 +45,43 @@ var _ = Describe("Network Binding", func() {
 		testBindingName2  = "binding2"
 		testSidecarImage2 = "image2"
 		testNetworkName3  = "net1"
+		testBindingName3  = "binding3"
 	)
 
+	Context("validation", func() {
+		It("should fail when unregistered binding plugin is being used", func() {
+			vmi := libvmi.New(
+				libvmi.WithInterface(v1.Interface{Name: testNetworkName1, Binding: &v1.PluginBinding{Name: testBindingName1}}), libvmi.WithNetwork(&v1.Network{Name: testNetworkName1}),
+				libvmi.WithInterface(v1.Interface{Name: testNetworkName2, Binding: &v1.PluginBinding{Name: testBindingName2}}), libvmi.WithNetwork(&v1.Network{Name: testNetworkName2}),
+				libvmi.WithInterface(v1.Interface{Name: testNetworkName3, Binding: &v1.PluginBinding{Name: testBindingName3}}), libvmi.WithNetwork(&v1.Network{Name: testNetworkName3}),
+			)
+			config := &v1.KubeVirtConfiguration{
+				NetworkConfiguration: &v1.NetworkConfiguration{
+					Binding: map[string]v1.InterfaceBindingPlugin{testBindingName2: {}},
+				},
+			}
+			Expect(netbinding.ValidateVMINetBindingPlugins(vmi, config)).ToNot(Succeed())
```

Comment:
Please also verify the event was sent.

Reply:
It means that the event emitting has to be part of `ValidateVMINetBindingPlugins`.
There is no login in the event emitting its like a log message and cannot fail (return no error), testing it may end up fragile if it is expected to assert the event message (as it could change and the test will break).

Does it make sense to consolidate it in `ValidateVMINetBindingPlugins`?